server/src/models/user/index.ts

```diff
@@ -20,4 +21,7 @@ const userSchema = new Schema(
 
 const UserModel = mongoose.models.User || mongoose.model("User", userSchema);
 
-export default UserModel;
\ No newline at end of file
+export default UserModel;
```

Comment:
"Please note that 'recoveryCode' and 'resetCode' are mandatory fields that must be generated while creating a new user. Additionally, both 'assignTask' and 'createTask' will be an array of strings as they contain the ID of the assigned or created task by the user."